woocommerce/woocommerce-gateway-payfast
# Bump Node and NPM versions

### Description
This PR contains changes to bump node, npm, and composer versions.

Node: **>=16.13.0**
NPM: **>=8.0.0**

### Steps to Test
1. Run `nvm use; npm install; npm run build`
2. Make sure the build zip is generated properly and not contains unwanted folders and files.
3. Smoke test with generated zip.

### Documentation





### Changelog Entry

> Dev - Update node version from 12.0.0 to 16.13.0.
> Dev - Update npm version from 6.9.0 to 8.0.0.
